config/initializers/county_to_region_code.rb

```diff
@@ -0,0 +1,561 @@
+# Taken from https://gist.github.com/radiac/d91d2ed1b971c03d49e9b7bd85e23f1c
+UK_COUNTIES_AND_REGIONS = [
+  ["Guernsey", "Crown Dependencies"],
+  ["IOM", "Crown Dependencies"],
+  ["Jersey", "Crown Dependencies"],
+  ["North East Derbyshire", "East Midlands"],
+  ["Amber Valley", "East Midlands"],
+  ["Ashfield", "East Midlands"],
+  ["Bassetlaw", "East Midlands"],
+  ["Blaby", "East Midlands"],
+  ["Bolsover", "East Midlands"],
+  ["Boston", "East Midlands"],
+  ["Broxtowe", "East Midlands"],
+  ["Charnwood", "East Midlands"],
+  ["Chesterfield", "East Midlands"],
+  ["Corby", "East Midlands"],
+  ["Daventry", "East Midlands"],
+  ["Derby", "East Midlands"],
+  ["Derbyshire", "East Midlands"],
+  ["Derbyshire", "East Midlands"],
+  ["Derbyshire Dales", "East Midlands"],
+  ["East Lindsey", "East Midlands"],
+  ["East Northamptonshire", "East Midlands"],
+  ["Erewash", "East Midlands"],
+  ["Gedling", "East Midlands"],
+  ["Harborough", "East Midlands"],
+  ["High Peak", "East Midlands"],
+  ["Hinckley and Bosworth", "East Midlands"],
+  ["Kettering", "East Midlands"],
+  ["Leicester", "East Midlands"],
+  ["Leicestershire", "East Midlands"],
+  ["Leicestershire", "East Midlands"],
+  ["Lincoln", "East Midlands"],
+  ["Lincolnshire", "East Midlands"],
+  ["Lincolnshire", "East Midlands"],
+  ["Mansfield", "East Midlands"],
+  ["Melton", "East Midlands"],
+  ["Newark and Sherwood", "East Midlands"],
+  ["North Kesteven", "East Midlands"],
+  ["North West Leicestershire", "East Midlands"],
+  ["Northampton", "East Midlands"],
+  ["Northamptonshire", "East Midlands"],
+  ["Nottingham", "East Midlands"],
+  ["Nottinghamshire", "East Midlands"],
+  ["Nottinghamshire", "East Midlands"],
+  ["Oadby and Wigston", "East Midlands"],
+  ["Rushcliffe", "East Midlands"],
+  ["Rutland", "East Midlands"],
+  ["South Derbyshire", "East Midlands"],
+  ["South Holland", "East Midlands"],
+  ["South Kesteven", "East Midlands"],
+  ["South Northamptonshire", "East Midlands"],
+  ["Wellingborough", "East Midlands"],
+  ["West Lindsey", "East Midlands"],
+  ["Babergh", "East of England"],
+  ["Basildon", "East of England"],
+  ["Bedford", "East of England"],
+  ["Bedford Borough", "East of England"],
+  ["Bedfordshire", "East of England"],
+  ["Braintree", "East of England"],
+  ["Breckland", "East of England"],
+  ["Brentwood", "East of England"],
+  ["Broadland", "East of England"],
+  ["Broxbourne", "East of England"],
+  ["Cambridge", "East of England"],
+  ["Cambridgeshire", "East of England"],
+  ["Cambridgeshire", "East of England"],
+  ["Castle Point", "East of England"],
+  ["Central Bedfordshire", "East of England"],
+  ["Chelmsford", "East of England"],
+  ["Colchester", "East of England"],
+  ["Dacorum", "East of England"],
+  ["East Cambridgeshire", "East of England"],
+  ["East Hertfordshire", "East of England"],
+  ["Epping Forest", "East of England"],
+  ["Essex", "East of England"],
+  ["Essex", "East of England"],
```

Comment:
Duplicate 😬

Reply:
There's a `.uniq` later in the file to handle these; should I just delete them at the source instead? I opted not to because doing it with code is more explicit and less error-prone.